code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -103,14 +106,16 @@
 	if (active)
 		force = 30
 		w_class = W_CLASS_LARGE
-		sharpness = 1.5
+		sharpness = initial(sharpness)
```

Comment:
God I hate this use of `initial()`
I know we use it for similar things elsewhere, but it fucks with VV and is just troublesome in general and we shouldn't.
I recommend storing the numbers for its on and off states and reading from those when switched instead.